jcl/src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -391,8 +391,17 @@ public StackTraceElement toStackTraceElement() {
 					moduleVersion = versionInfo.get().toString();
 				}
 			}
-			return new StackTraceElement(classLoaderName, moduleName, moduleVersion, className, methodName, fileName,
+
+			StackTraceElement element = new StackTraceElement(classLoaderName, moduleName, moduleVersion, className, methodName, fileName,
 					lineNumber);
+
+			/**
+			 * Disable including classloader name and module version in stack trace output
+			 * until StackWalker StackTraceElement include info flags can be set properly
+			 */
+			element.disableIncludeInfoFlags();
+
+			return element;
```

Comment:
@pshipton Here is the temporary change to `StackFrame` so that the existing behaviour does not change. `disableIncludeInfoFlags()` was added to `StackTraceElement` to disable the include flags for classloader name and module version. I added a note to the end of the PR description that `StackFrame` stack trace output will be fixed in the future.